doc/testing-your-application.rst

```diff
@@ -121,6 +121,56 @@ For the `assertHit` and `assertMiss` assertions to work, you need to add a
 :ref:`custom X-Cache header <nginx_debugging>` to responses served
 by your Nginx.
 
+SymfonyTestCase
+---------------
+
+The ``SymfonyTestCase`` does not control a web server. You will need to run
+your own web server with your application on the test system. The test case
+will help you to send requests to the cache and to invalidate the cache folder
+of the Symfony HttpCache.
```

Comment:
Maybe begin with what it *does* do? Especially since it’s rather obvious one needs some Apache/Nginx to run Symfony on.